cfgov/unprocessed/css/post.less

```diff
@@ -384,31 +377,17 @@
     });
 }
 
+// TODO: `post_dek` is only used on events/_event-detail.html.
+//       Refactor to use CF classes.
+
 .post_dek {
     .h4();
     // Eyeballed to 30px.
     margin-bottom: unit(20px / @font-size, em);
 }
 
-.post_meta {
-    margin-bottom: unit(18px / @base-font-size-px, em);
-}
-
-.post_byline {
-    .h4();
-}
-
-.post_featured-img {
-    display: block;
-    margin: unit(16px / @base-font-size-px, em) 0;
-    .respond-to-min(@bp-sm-min, {
-        // 20px value eyeballed to 24px.
-        margin:
-            unit(30px / @base-font-size-px, em)
-            0
-            unit(20px / @base-font-size-px, em);
-    });
-}
+// TODO: `post_body` is used on article.html and events/_event-detail.html.
+//       Refactor to use CF classes if possible.
 
 .post_body {
```

Comment:
It's currently adding some margin on the individual block pages. Definitely can be refactored, but for this PR I'm trying to remove things not refactor them.